src/AGPL-3.0.xml

```diff
@@ -1,580 +1,848 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <SPDXLicenseCollection xmlns="http://www.spdx.org/license">
-   <license isOsiApproved="true" licenseId="AGPL-3.0"
-            name="GNU Affero General Public License v3.0">
-      <crossRefs>
-         <crossRef>http://www.gnu.org/licenses/agpl.txt</crossRef>
-         <crossRef>http://www.opensource.org/licenses/AGPL-3.0</crossRef>
-      </crossRefs>
-      <standardLicenseHeader> Copyright (C) 
-    <alt match=".+" name="copyright">[year] [name of author]</alt> This program is free software: you can
-    redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the
-    Free Software Foundation, version 3. This program is distributed in the hope that it will be useful, but WITHOUT
-    ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-    GNU Affero General Public License for more details. You should have received a copy of the GNU Affero General
-    Public License along with this program. If not, see &lt;http://www.gnu.org/licenses/&gt;
-  </standardLicenseHeader>
-      <notes>This version was released: 19 November 2007</notes>
-      <titleText>
-         <p>GNU AFFERO GENERAL PUBLIC LICENSE 
-        <br/>Version 3, 19 November 2007 
+  <license licenseId="AGPL-3.0" isOsiApproved="true"
+  name="GNU Affero General Public License v3.0">
+    <crossRefs>
+      <crossRef>http://www.gnu.org/licenses/agpl.txt</crossRef>
+      <crossRef>http://www.opensource.org/licenses/AGPL-3.0</crossRef>
+    </crossRefs>
+    <standardLicenseHeader>
+      Copyright (C)<alt name="copyright" match=".+">[year] [name of author]</alt>
```

Comment:
Removing the space between `(C)` and `<alt` is a bug in your pretty-printer (see 7af2b40, #526, fixing another instance of this issue).